packages/composites/bob/README.md

```diff
@@ -0,0 +1,71 @@
+# Bob adapter for Chainlink
+
+The adapter provides an interface for retrieving data from the Bitcoin blockchain.
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+| Required? |   Name    |       Description       | Options |       Defaults to       |
+| :-------: | :-------: | :---------------------: | :-----: | :---------------------: |
+|           | `RPC_URL` | Blockchain RPC endpoint |         | `http://localhost:8545` |
+
+Other common `RPC_URL` endpoints:
+
+- bitcoind: http://localhost:8332
+- btcd: http://localhost:8334
+
+
+This composite adapter incorporates the [`json-rpc`](../../sources/json-rpc) adapter by calling `"method": "getblockchainfo"`.
+
+## Running
+
+See the [Composite Adapter README](../README.md) for more information on how to get started.
+
+- `endpoint`: The parameter to query for. Default: "difficulty"
```

Comment:
Since there is more than one endpoint already, it would be good to document them all here.

This more frequently happens in source adapters, an example that can be used as a template:
https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/sources/dxfeed

Reply:
Yes, I believe we are expecting to add more endpoints in the future, or at least allowing for that possibility. I'll go ahead and add some documentation to the README. Thanks!